packages/app-desktop/gui/NoteListControls/NoteListControls.tsx

```diff
@@ -116,7 +126,32 @@ function NoteListControls(props: Props) {
 		if (!props.showNewNoteButtons) return null;
 
 		return (
-			<ButtonContainer>
+			<RowContainer>
+				<StyleNewTodoButton
+					className="new-todo-button"
+					tooltip={CommandService.instance().label('newTodo')}
+					title="+ New to do"
```

Comment:
that should be translated, so something like `_('+ %s', 'New to-do')`. And in the UI we generally use the term "to-do" with a dash

Reply:
Fixed.